cmd/minikube/cmd/status.go

```diff
@@ -106,7 +106,7 @@ var statusCmd = &cobra.Command{
 
 		var statuses []*Status
 		for _, n := range cc.Nodes {
-			glog.Infof("checking status of %s ...", n.Name)
+			glog.Infof("checking status of %s ...", cname)
```

Comment:
I think the purpose of this log is to print info about each node in the loop, instead of the entire cluster. Could you please move this log one line down and print out the value of `machineName` instead of `cname`?

Reply:
OK I see, now it makes more sense :)